README.md

```diff
@@ -2,86 +2,85 @@
+* `serializer` (function, default `el => el.innerHTML`) controls how the the `content` property returned by the `parse()` method is produced from the root DOM element. It may be useful to specify the `serializer` as the identity function (`el => el`) to obtain a DOM element instead of a string for `content` if you plan to process it further.
 
-```
+### `parse()`
+
+Returns an object containing the following properties:
+
+* `title`: article title;
+* `content`: HTML string of processed article content;
+* `textContent`: text content of the article, with all the HTML tags removed;
+* `length`: length of an article, in characters;
+* `excerpt`: article description, or short excerpt from the content;
+* `byline`: author metadata;
+* `dir`: content direction;
+* `siteName`: name of the site.
+
+The `parse()` method works by modifying the DOM. This removes some elements in the web page, which may be undesirable. You can avoid this by passing the clone of the `document` object to the `Readability` constructor:
+
+```js
 var documentClone = document.cloneNode(true); 
 var article = new Readability(documentClone).parse();
 ```
 
-## Usage from Node.js
+### `isProbablyReaderable(document, options)`
+
+A quick-and-dirty way of figuring out if it's plausible that the contents of a given document are suitable for processing with Readability. It is likely to produce both false positives and false negatives. The reason it exists is to avoid bogging down a time-sensitive process (like loading and showing the user a webpage) with the complex logic in the core of Readability. Improvements to its logic (while not deteriorating its performance) are very welcome.
+
+## Node.js usage
 
 Readability is available on npm:
 
 ```bash
 npm install @mozilla/readability
 ```
 
-In Node.js, you won't generally have a DOM document object. To obtain one, you can use external
-libraries like [jsdom](https://github.com/jsdom/jsdom). While this repository contains a parser of
-its own (`JSDOMParser`), that is restricted to reading XML-compatible markup and therefore we do
-not recommend it for general use.
-
-If you're using `jsdom` to create a DOM object, you should ensure that the page doesn't run (page)
-scripts (avoid fetching remote resources etc.) as well as passing it the page's URI as the `url`
-property of the `options` object you pass the `JSDOM` constructor.
-
-### Example:
+Since Node.js does not come with its own DOM implementation, we rely on external libraries like [jsdom](https://github.com/jsdom/jsdom). Here's an example using `jsdom` to obtain a DOM document object:
 
 ```js
 var { Readability } = require('@mozilla/readability');
-var JSDOM = require('jsdom').JSDOM;
-var doc = new JSDOM("<body>Here's a bunch of text</body>", {
+var { JSDOM } = require('jsdom');
+var doc = new JSDOM("<body>Look at this cat: <img src='./cat.jpg'></body>", {
```

Comment:
Added an `<img>` with a relative `src` to tie to the next paragraph, explaining the purpose of the `url` option in JSDOM.